### DISCUSSION TITLE
submission of false positive or not

### DISCUSSION THREAD

**Discussion by nulkathos** [blakeblackshear/frigate#10236] [General]
I've a question on what to do in some situation

![image](https://github.com/blakeblackshear/frigate/assets/155397425/61f61d2a-03ec-4692-af1b-12bff3afb333)

it's a false positive because the box is far big than the object ? if yes, we submit to frigate+ and label needs to be created of the car with a box with right size.

or...it's true ? if is true, can be submitted and on frigate+ left asis or can be the box resized ??

**Comment by NickM-27** [comment]
this is a bit subjective, regardless if you choose true or false it won't have a big impact. In this case I would recommend selecting false positive then labeling the car

**Reply by nulkathos** [reply]
and snapshot like that

![image](https://github.com/blakeblackshear/frigate/assets/155397425/ba170724-454f-44e5-8a76-c53878a47b31)

it's a car yes....but the image I think is not good ... what is suggested to do ? only marked as a false positive and submitted or marked as a false positive and mark as a car fixing the box to the real size ?

**Reply by nulkathos** [reply]
I'm trying to understand what is the right way in some complicated situations

**Reply by NickM-27** [reply]
I mean I definitely would not call that a false positive, it is a car and the box is quite close
